ui/src/components/page_navigation/PageNavigation.js

```diff
@@ -85,7 +85,6 @@ const MoreActionsButton = ({ actions }) => {
       panelPaddingSize="none"
       anchorPosition="downRight">
       <EuiContextMenuPanel
-        hasFocus={false}
```

Comment:
No longer a valid prop, this throws a console error if set: https://elastic.github.io/eui/#/navigation/context-menu